_includes/headtags.html

```diff
@@ -2,10 +2,11 @@
 {% for lng in site.languages %}
     {% if lng == site.default_lang %}
       {% capture langlink %}{{ site.a_rootpage }}{{ page.url }}{% endcapture %}
+          <link rel="canonical" hreflang="{{ lng }}" href="{{ langlink }}">
```

Comment:
Should we have `canonical` on _every_ page and the `alternate` only on the languages?  Otherwise how does Google match the alternates to the canonical?

Reply:
We do (with this change), but is that not what you mean?

<img width="395" alt="image" src="https://github.com/jamulussoftware/jamuluswebsite/assets/4561747/77bb609a-750c-4d4d-a067-71d3ae418d8b">

Reply:
Ah, I'm misreading how the loop works... 👍🏻